notebooks/standardize_events_data.ipynb

```diff
@@ -449,21 +459,18 @@
     "    }\n",
     "}\n",
     "\n",
-    "def get_event_type(token_address, key_name):\n",
-    "    token_type = TOKEN_MAPPING.get(token_address.lower())\n",
+    "def get_nostra_alpha_event_type(token_address: str, key_name: str) ->str:\n",
+    "    token_type = NOSTRA_ALPHA_TOKENS_TO_TOKEN_TYPES.get(token_address.lower())\n",
     "    if token_type:\n",
-    "        return EVENT_MAPPING.get(token_type, {}).get(key_name)\n",
-    "    return 'interest rate updated'  # Default to 'interest rate updated' if not matched\n",
+    "        return NOSTRA_ALPHA_TOKEN_TYPES_TO_EVENT_TYPES.get(token_type, {}).get(key_name)\n",
+    "    return 'interest_rate_updated'  # Default to 'interest rate updated' if not matched\n",
```

Comment:
The default should be raising an error (if no match is found). An `interest_rate_updated` should be returned when `token_address` is `0x03d39f7248fb2bfb960275746470f7fb470317350ad8656249ec66067559e892` and `key_name` is `InterestStateUpdated`.